docs/REST API/rest-api.md

```diff
@@ -9173,6 +9173,12 @@ Returns information about the specified software. By default, `versions` are sor
       "latest_version": "2.04",
       "icon_url": "https://is1-ssl.mzstatic.com/image/thumb/Purple211/v4/f1/65/1e/a4844ccd-486d-455f-bb31-67336fe46b14/AppIcon-1x_U007emarketing-0-7-0-85-220-0.png/512x512bb.jpg",
       "self_service": true,
+      "automatic_install_policies": [
+        {
+          "policy_id": 345,
+          "policy_name": "[Install software] Logic Pro",
```

Comment:
I'd suggest using the same key names as policies (so it's the same data, just a simplified version)
```suggestion
          "id": 345,
          "name": "[Install software] Logic Pro",
```